pkgs/top-level/perl-packages.nix

```diff
@@ -13120,6 +13120,16 @@ let
       url = mirror://cpan/authors/id/A/AB/ABH/Mozilla-CA-20180117.tar.gz;
       sha256 = "f2cc9fbe119f756313f321e0d9f1fac0859f8f154ac9d75b1a264c1afdf4e406";
     };
+
+    buildInputs = [ pkgs.cacert ];
+
+    postPatch = ''
+      rm lib/Mozilla/CA/cacert.pem
+      rmdir lib/Mozilla/CA
+      substituteInPlace lib/Mozilla/CA.pm \
+        --replace 'return $file;' 'return "${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt";';
+    '';
```

Comment:
maybe just create a symlink
`lib/Mozilla/CA/cacert.pem` ->` ${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt` ?

```
ln -s --force ${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt lib/Mozilla/CA/cacert.pem
```

or does it need a real file?

Reply:
Agree, this is a better solution.

Since symlinks are canonicalized in used /nix/store, I updated the PR to use `cp` instead. Not sure what is preferred.

Reply:
I am not sure what you are mean by `canonicalized`. @conferno solution looks ok to me.

Reply:
Sorry, turned into a hard link.

Fixed.